### ISSUE TITLE
[A11y] screen reader cannot announce and select items of ListView() on destop app for Windows

### ISSUE THREAD

**Issue by yama3nomori** [flutter/flutter#98376]
On Windows app, screen reader cannot focus ListView().
So user cannot select items from that.

```
[√] Flutter (Channel beta, 2.10.0-0.1.pre, on Microsoft Windows [Version 10.0.22000.434], locale ja-JP)
    • Flutter version 2.10.0-0.1.pre at C:\Android\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 628f0e3f3a (2 days ago), 2022-01-11 19:50:05 -0600
    • Engine revision 63ca99584a
    • Dart version 2.16.0 (build 2.16.0-134.1.beta)
    • DevTools version 2.9.2

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Android\sdk
    X cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[X] Visual Studio - develop for Windows
    X Visual Studio not installed; this is necessary for Windows development.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.63.2)
    • VS Code at C:\Users\nyama\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.26.0

[√] Connected device (4 available)
    • S1 (mobile)       • MS3GLP6711409145 • android-arm64  • Android 9 (API 28)
    • Windows (desktop) • windows          • windows-x64    • Microsoft Windows [Version 10.0.22000.434]
    • Chrome (web)      • chrome           • web-javascript • Google Chrome 96.0.4664.110
    • Edge (web)        • edge             • web-javascript • Microsoft Edge 96.0.1054.62

[√] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 2 categories.
```

**Comment by darshankawar** [comment]
@yama3nomori 
Check [this](https://github.com/flutter/flutter/issues/94791#issuecomment-996139061) issue comment. According to it, can you switch to latest master and try your use case once again and see if it works as expected or still the behavior persists ?
